corehq/tests/nose.py

```diff
@@ -26,7 +26,11 @@
 from nose.plugins import Plugin
 from django.apps import AppConfig
 from django.conf import settings
-from django.db.backends.creation import TEST_DATABASE_PREFIX
+try:
+    from django.db.backends.base.creation import TEST_DATABASE_PREFIX
+except ImportError:
+    # TODO - remove when django >= 1.8
+    from django.db.backends.creation import TEST_DATABASE_PREFIX
```

Comment:
do we also need this in [settingshelper.py](https://github.com/dimagi/commcare-hq/blob/a57fdc1ec617e75ab1d66114a41241ad74be5efe/settingshelper.py#L7)?